Backpack/Dialog/Classes/BPKDialogView.m

```diff
@@ -45,6 +45,8 @@ @interface BPKDialogView ()
 @property(nonatomic, strong) BPKLabel *titleLabel;
 @property(nonatomic, strong) BPKLabel *descriptionLabel;
 @property(nonatomic, strong) UIStackView *buttonStackView;
+@property(nonatomic, strong) UIColor *iconBackgroundColorValue;
```

Comment:
I'm not sure if using these underlying properties is the best way to respond to them being set... 🤔

Reply:
Agreed. If I understand correctly we should be able to call `setIconBackgroundColor` directly without adding state.

Reply:
Think I'll refactor this to use another object, so that consumers must provide neither or both 👍